Assets/Scripts/Combat/Combantants/Combatant.cs

```diff
@@ -14,16 +14,31 @@ public abstract class Combatant : MonoBehaviour
     public int CurrentHealthPoints;
     public int MaxShieldPoints;
     public int CurrentShieldPoints;
-    private readonly float HealthBarPosition = 125f;
     protected CombatSystem CombatSystem;
     [SerializeField]
     GameObject HealthBarPrefab;
     [SerializeField]
+    GameObject ShieldBarPrefab;
+    [SerializeField]
     GameObject HealthBarUIPanel;
     [SerializeField]
     GameObject Shield;
 
     private GameObject HealthBar;
+    private HealthBar RedBar;
+    private Text HealthText;
+
+    private bool IsShieldActive;
+    private GameObject ShieldBar;
+    private HealthBar BlueBar;
+    private Text ShieldText;
+
+    private const float HealthBarYOffset = 1.45f;
+    private const float ShieldBarYOffset = 2f;
+
+    // Temporary fix while I wait for a collider to be added
+    private const float HealthBarPosition = 125f;
+    private const float ShieldBarPosition = 145f;
```

Comment:
From my understanding, it seems that Sield doesn't have a collider because there is one included in the character controller that's being used. Maybe that's something you could use?

Reply:
I have no idea, will try to look into that

Reply:
CharacterController is a child of the Collider class. I think you can still reference the Collider from it!

Reply:
https://docs.unity3d.com/ScriptReference/CharacterController.html
Check under the title

Reply:
The problem I'm encountering is that the bounds value is not correct and it's set to 0